packages/babel-helpers/src/helpers.js

```diff
@@ -1011,3 +1011,12 @@ helpers.classPrivateFieldSet = () => template.program.ast`
     return value;
   }
 `;
+
+helpers.classStaticPrivateFieldBase = () => template.program.ast`
+  export default function _classStaticPrivateFieldBase(receiver, classConstructor, privateClass) {
```

Comment:
So there are a few issues here, and we can choose either of two ways to handle it:

- We can remove this `privateClass` holder object for `loose` mode.
- We can handle `get`, `set`, and `call` for both `loose` and `spec` mode.

The first option would allow us to reuse the same code we already have for instance private fields.

If we continue to use a holder object, we have to handle `get`, `set`, and `call` operations on the holder, and use the `receiver` as the calling context for all those cases. This complicates the runtime for `loose` mode. But, we're gonna need to do that anyways for spec mode.

Reply:
This is super helpful, I had a hard time understanding the line between loose and spec.